tests_integration/nativeapp/test_templates.py

```diff
@@ -1,12 +1,12 @@
 import pytest
-from snowflake.connector.compat import IS_WINDOWS
 
 
 @pytest.mark.integration
-@pytest.mark.skipif(IS_WINDOWS, reason="Permissions issue on Windows")
-def test_list_templates_no_options_success(runner, snapshot):
+def test_list_templates_no_options_success(runner):
     args = ["app", "list-templates"]
-    result = runner.invoke(args)
+    result = runner.invoke_json(args)
 
     assert result.exit_code == 0
-    assert result.output == snapshot
+    templates = result.json
+    assert len(templates) > 0
+    assert "basic" in [t["template"] for t in templates]
```

Comment:
The snapshot test was failing due to a different number of columns. In general I don't think updating the README or adding more templates to the repo should cause the tests to fail, so I reduced the checks to just making sure there's a basic template in the output. Thoughts?

Reply:
The change seems reasonable, but I can imagine a bug that only returns the first template. I'd be more comfortable with asserting `streamlit-python` and `streamlit-java` are there too, though I don't think we need to be exhaustive here as we add more templates.